extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -120,6 +131,32 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
     if (!this.isOnActivePage_() || !isOpen) {
       return;
     }
+
```

Comment:
The comment above "This happens when the "Shop Now" CTA is clicked" is outdated if this is called when clicking on the tag as well. Pls update